### PR TITLE
Fix all code inspection warnings.

### PR DESCRIPTION
**Pull request by jkmssoft**

### CODE REVIEW

File: views/_alert.php [hunk 1]
```diff
@@ -16,14 +16,17 @@
...
+                <?php if (in_array($type, ['success', 'danger', 'warning', 'info'])) : ?>
+                    <?= Alert::widget([
+                            'options' => ['class' => 'alert-dismissible alert-'.$type],
+                            'body' => $message
+                    ]) ?>
                 <?php endif ?>
             <?php endforeach ?>
         </div>
     </div>
-<?php endif ?>
```
Comment (dmeroff) [inline views/_alert.php]: This `?>` removal should be reverted (in all view files below). It just does not make any sense.

File: views/admin/_info.php [hunk 2]
```diff
@@ -22,28 +22,37 @@
         <td><strong><?= Yii::t('user', 'Registration time') ?>:</strong></td>
         <td><?= Yii::t('user', '{0, date, MMMM dd, YYYY HH:mm}', [$user->created_at]) ?></td>
     </tr>
-    <?php if ($user->registration_ip !== null): ?>
+    <?php if ($user->registration_ip !== null) : ?>
```
Comment (dmeroff) [inline views/admin/_info.php]: This should also be reverted.

**Comment by dmeroff** [comment]
Thank you, but this PR has some problems which should be solved before merging. I've added comments to your commits.

File: views/_alert.php (hunk 1 above)
Comment (jkmssoft) [inline views/_alert.php]: All other php files do not end with a `?>` so I thought view files should also not end with a `?>`.
See commits below, I have changed it for you :)

**Comment by jkmssoft** [comment]
Please let me know, if there are any other issues with this PR.

**Comment by dmeroff** [comment]
It is ok now. Thank you!

**Pull request merged**